scripts/release/controller/build.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+echo
+date "+build_release begin BUILD stage %Y%m%d_%H%M%S"
+echo
+
+set -ex
+
+export GOPATH=${HOME}/go
+export PATH=${HOME}/gpgbin:${GOPATH}/bin:/usr/local/go/bin:${PATH}
+
+# Anchor our repo root reference location
+REPO_ROOT="${HOME}"/go/src/github.com/algorand/go-algorand/
+
+cd "${REPO_ROOT}"
+export RELEASE_GENESIS_PROCESS=true
+export HASH="$1"
+export CHANNEL="$2"
+PLATFORM=$("${REPO_ROOT}"/scripts/osarchtype.sh)
+PLATFORM_SPLIT=(${PLATFORM//\// })
+OS=${PLATFORM_SPLIT[0]}
+ARCH=${PLATFORM_SPLIT[1]}
+DEFAULTNETWORK=$(PATH=${PATH} "${REPO_ROOT}"/scripts/compute_branch_network.sh)
+export DEFAULTNETWORK
+export PKG_ROOT=${HOME}/node_pkg
+export VARIATIONS="base"
+# tell underlying 'build' scripts we already built
```

Comment:
this comment was meant to go with `export NO_BUILD=true`

Reply:
I'll fix.